nyc_taxi/nyc_taxi.ipynb

```diff
@@ -360,6 +369,8 @@
     "\n",
```

Comment:
I think this code can be adapted to no longer use `shade()`. Please try this out:

```
def transform(img):
    agg = img.data["dropoff_x_dropoff_y Count"]
    return img.clone(agg.where(agg>np.percentile(agg, 90)))

custom_points = hv.Points(df, ['dropoff_x', 'dropoff_y'])
custom_rasterized = hd.rasterize(custom_points).apply(transform).opts(cmap='fire', cnorm='eq_hist', tools=['hover'])
tiles = EsriStreet().redim.range(x=x_range, y=y_range).opts(alpha=0.5)

tiles * hd.dynspread(custom_rasterized, threshold=0.3, max_px=4)
```

Reply:
Nice!